modules/bls-dkg/package.json

```diff
@@ -7,7 +7,7 @@
     "unit-test": "mocha",
     "test": "npm run unit-test"
   },
-  "author": "John Driscoll <john@bitgo.com>",
+  "author": "BitGo Inc.",
```

Comment:
How about this?
```suggestion
  "author": "BitGo SDK Team <sdkteam@bitgo.com>",
```

Reply:
Looks good to me. I'll make the changes.